src/Cohesity.Powershell/Scripts/Restore/restore-cohesityoracledatabase.ps1

```diff
@@ -31,6 +31,9 @@ function Restore-CohesityOracleDatabase {
         # Specifies the Oracle base directory path.
         [string]$OracleBase,
         [Parameter(Mandatory = $true)]
+        # Location to put the database files(datafiles, logfiles etc.).
+        [string]$DatabaseFileDestination,
+        [Parameter(Mandatory = $true)]
```

Comment:
This willl break backward compatibility of the cmdlet ?

Reply:
nope, we just added new parameter. As this is an optional one, even if the user doesn't provide this, it will run successfully.

Reply:
Why is this a mandatory param? @KavishreeShanmugam11 
1. Is it relevant when recovering database to same location?

Reply:
This script has been created for alternate location only. So, they made parameters that are required for alternation location restore as mandatory. This script needs to be modified irrespective of restore location.

Reply:
> nope, we just added new parameter. As this is an optional one, even if the user doesn't provide this, it will run successfully.

Shouldn't it be : 
[Parameter(Mandatory = $false)]

Reply:
Done